src/main/java/com/terraformersmc/smolder/api/SmolderBiomeRegistry.java

```
@@ -185,6 +207,17 @@ public static SmolderBiome getParent(SmolderBiome biome) {
 		return PARENTS.get(biome);
 	}
 	
+	/**
+	 * Returns {@link SmolderBiome} attached to biome or null
+	 * @param biome
+	 * @return
+	 */
+	public static SmolderBiome getFromBiome(Biome biome)
+	{
+		Optional<RegistryKey<Biome>> optional = BuiltinRegistries.BIOME.getKey(biome);
+		return optional.isPresent() ? LINKS.get(optional.get()) : null;

```

Not sure what you're saving here with the optional since you're returning null anyways. I thin this could be replaced it a getOrElse instead of a ternary statement if you want to keep it.

---

Biome registry can return only optional for biome key. Using this check "No value present" error will be ignored